tests/rptest/tests/wasm_failure_recovery_test.py

```diff
@@ -1,81 +0,0 @@
-# Copyright 2021 Vectorized, Inc.
-#
-# Use of this software is governed by the Business Source License
-# included in the file licenses/BSL.md
-#
-# As of the Change Date specified in that file, in accordance with
-# the Business Source License, use of this software will be governed
-# by the Apache License, Version 2.0
-
-import random
-from rptest.clients.types import TopicSpec
-from rptest.tests.wasm_identity_test import WasmIdentityTest
-from rptest.wasm.topics_result_set import materialized_result_set_compare
-
-
-class WasmFailureRecoveryTest(WasmIdentityTest):
```

Comment:
I'm good with removing this while wasm is tech preview, but let's think about adding the coverage of "wasm_engine down, but redpanda is up" scenarios back when transitioning to GA in the future.